tests/foreman/cli/test_remoteexecution.py

```diff
@@ -943,3 +949,123 @@ def test_positive_install_ansible_collection(self, fixture_sca_vmsetup, module_g
         assert result['success'] == '1'
         collection_path = str(client.execute('ls /etc/ansible/collections/ansible_collections'))
         assert 'oasis' in collection_path
+
+
+class TestRexUsers:
+    """Tests related to remote execution users"""
+
+    @pytest.fixture(scope='module')
+    def module_rexmanager_user(self, module_org):
+        """Creates a user with Remote Execution Manager role"""
+        password = gen_string('alpha')
+        rexmanager = gen_string('alpha')
+        make_user({'login': rexmanager, 'password': password, 'organization-ids': module_org.id})
```

Comment:
I won't block on this, but fixtures should be executed through nailgun API entities, moving away from cli factory functions.